Prep release (#146)
twilio/twilio-live-interactive-video: CHANGELOG.md

# 1.1.0 (July 11, 2022)

### New Feature

- This release adds a chat feature for the host, speakers, and viewers. This feature allows all users to send and receive textual messages to each other while connected to a stream. This feature is powered by the [Twilio Conversations API](https://www.twilio.com/conversations-api) and is optional. See the [README.md](https://github.com/twilio/twilio-live-interactive-video/blob/feature/audience-chat/README.md#set-your-account-sid-and-auth-token) for more information on how to opt out.

### Bug Fixes

> - Fixes an issue where the host was erroneously creating more than one stream at a time. [#116](https://github.com/twilio/twilio-live-interactive-video/pull/116)

Reviewer: ```suggestion
- Fixes an issue where the host could not create more than one stream. [#116](https://github.com/twilio/twilio-live-interactive-video/pull/116)
```
